### PR TITLE
Fixes 8558 - Support installation of foreman-digitalocean plugin

### PR DESCRIPTION
**Pull request by dLobatog**
Linked to https://github.com/theforeman/puppet-foreman/pull/258

### CODE REVIEW

File: config/answers.yaml [hunk 1]
```diff
@@ -20,6 +20,7 @@ foreman::plugin::bootdisk: true
 foreman::plugin::chef: false
 foreman::plugin::default_hostgroup: false
 foreman::plugin::discovery: false
```
Comment (domcleal) [inline config/answers.yaml]: swap these lines :)

File: config/foreman-installer.yaml [hunk 2]
```diff
@@ -52,6 +52,9 @@
     :dir_name: foreman
     :manifest_name: plugin/discovery
     :params_name: plugin/discovery/params
+  :foreman::plugin::digitalocean:
```
Comment (domcleal) [inline config/foreman-installer.yaml]: ditto

**Comment by domcleal** [comment]
:+1: untested, will do later.

Missing '#' in the commit message too.

**Comment by dLobatog** [comment]
@domcleal Done, how come prprocessor didn't catch my mistake in the commit title?

**Comment by domcleal** [comment]
We don't have it enabled on the installer repos as they don't use Jenkins for testing and don't tend to use redmine.

**Comment by domcleal** [comment]
Merged as 6ad8e62, thanks @eLobato.

**Pull request closed without merging**